homeassistant/components/camera/xeoma.py

```diff
@@ -86,7 +86,7 @@ def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
                          camera[CONF_VIEWER_USERNAME],
                          camera[CONF_VIEWER_PASSWORD]) for camera in cameras])
     except XeomaError as err:
-        _LOGGER.error("Error: %s", err.message)
+        _LOGGER.error("Error: %s", err.message)  # noqa: B306
```

Comment:
I'm confused why we would disable this and not just fix?

Reply:
I don't see anything to fix offhand. XeomaError's message really is in its `message` attribute: https://github.com/jeradM/pyxeoma/blob/master/pyxeoma/xeoma.py#L139

The other few B306 cases in mercedesme and tesla are similar.

Reply:
Ah I see. In that case I think that it's better to disable this check too. We don't want lint rules to "guess" what is wrong and generate false positives. This is something that every new developer will get wrong and thus we can't have that.

Reply:
Going through the list of bugbear, it doesn't seem like it adds too much value. I think we can just stick with flake8 and pylint.

Reply:
I disagree. For example B006 is a very useful, if not essential check to have. B008 is good too, although it is prone to false positives. B002, B003, B004, B007 are nice to have as well. If you want, I can take a look at reworking the patch to enable only these from bugbear.